docs/develop/dapps/ton-connect/message-builders.mdx

```diff
@@ -701,7 +701,7 @@ import { beginCell, toNano } from '@ton/ton'
         .storeAddress(Address.parse(NEW_OWNER_WALLET)) // new_owner:MsgAddress
         .storeAddress(Address.parse(Wallet_DST))       // response_destination:MsgAddress
         .storeUint(0, 1)                         // custom_payload:(Maybe ^Cell)
-        .storeCoins(toNano(0.000000001).toString())       // forward_amount:(VarUInteger 16)
+        .storeCoins(toNano("0.01"))              // forward_amount:(VarUInteger 16)
```

Comment:
We need to keep 1 nanoTon here, as we want to send a message with the minimal amount.